marklogic-data-hub-central/ui/src/pages/Run.test.tsx

```diff
@@ -631,7 +631,8 @@ describe('Verify map/match/merge/master step failures in a flow', () => {
             fireEvent.click(exploreButton);
         }
         wait(() => {
```

Comment:
For my tests that follow in DHFPROD-5047, I need an await before wait, so:

`await wait(() => {`